rope/contrib/codeassist.py

```diff
@@ -347,15 +350,26 @@ def default_templates():
     return {}
 
 
+def _startswith(s1, s2):
```

Comment:
Code Smells in param names:
N1: Choose Descriptive Names
N4: Unambiguous Names